### ISSUE TITLE
Latest Paper related to DeViSE

### ISSUE THREAD

**Issue by UmarMajeed-Rana** [fg91/DeViSE-zero-shot-prediction#3]
Hi Fabio.
I read your article on Medium. Due to some reasons I am not able to post response. I enjoyed reading your explanation of Paper. Can you point recent advancements in this space? I see this Paper was published in 2013 but it still looks relevant. Simple and Powerful.

**Comment by fg91** [comment]
Hi Umar,

thank you for your interest! If you are interested in these kinds of models I suggest you next look at image caption generation:
https://cs.stanford.edu/people/karpathy/cvpr2015.pdf
The authors "describe neural networks that map words and image
regions into a common, multimodal embedding.", which shares some similarities with DeViSE. Better approaches are, however, presented in articles like these ([1](https://www.cv-foundation.org/openaccess/content_cvpr_2015/papers/Vinyals_Show_and_Tell_2015_CVPR_paper.pdf),[2](https://arxiv.org/pdf/1502.03044.pdf)).
I'm working on a detailed image caption generation tutorial that should be on Medium within the next few weeks.
Best regards,
Fabio

**Comment by UmarMajeed-Rana** [comment]
Thank you for your response Fabio. Do show and tell and other article you
mentioned they also take image and text into same common space? I am
interested in doing reverse image search on fashion domain. Searching
images from corpus by writing the description.

**Comment by fg91** [comment]
No, the models proposed in the "Show and Tell" and "Show, Attend and Tell" papers directly generate captions using an RNN decoder taking the representations generated by a CNN encoder as input.

However, the last paragraph in the left column in the section "Related Work" in the "Show and Tell" paper (page 2) talks about previous articles doing exactly what you intend to do as far as I understand. Maybe check the paragraph out (the one starting with "A large body of work has addressed the problem of ranking descriptions for a given image...").
Is this what you were looking for?

Best regards,

Fabio

**Comment by UmarMajeed-Rana** [comment]
Hi Fabio

Exactly I am looking some thing similar. I am looking for some latest paper
in this domain so that I can extend them with focus on fashion domain.
I saw many attention based models for image captioning but they are not in
common embedding space with text. At Inference Time I will just have text
to find the relevant/related image.
